File: benchmarks/evidence_reuse/fixtures/node_esm_fixture/tests/app.test.mjs
import assert from "node:assert/strict";
import test from "node:test";

import { compute, message, modeLabel } from "../src/app.mjs";

test("compute", () => {
  assert.equal(compute(7), 12);
});

test("message", () => {
  assert.equal(message("  Ada "), "Welcome, Ada!");
});

test("mode", () => {
  assert.equal(modeLabel(), "test");
});

File: benchmarks/evidence_reuse/fixtures/java_fixture/tests/AppTest.java
public final class AppTest {
    private AppTest() {}

    public static void main(String[] arguments) {
        assertEquals(8, App.compute(4));
        assertEquals("Welcome Ada", App.message("  Ada "));
        assertEquals("test", App.modeLabel());
    }

    private static void assertEquals(Object expected, Object actual) {
        if (!expected.equals(actual)) {
            throw new AssertionError("expected " + expected + " but got " + actual);
        }
    }
}
